src/main/java/pl/edu/agh/firecell/engine/algorithm/TemperaturePropagator.java

```diff
@@ -89,4 +96,7 @@ private static double tempDiff(Cell cellOne, Cell cellTwo) {
         return cellOne.temperature() - cellTwo.temperature();
     }
 
+    private static boolean isCellBurning(Cell cell) {
+        return cell.flammable() && cell.burningTime() > 0;
+    }
```

Comment:
This dunction is already defined in BasicAlgorithm

Reply:
And that's okay :P

Reply:
Why then not to use it? I don't undertand